### PR TITLE
Improve main file

### PR DESCRIPTION
**Pull request by szepeviktor**
@s3rgiosan Please see comments below.

### CODE REVIEW

File: date-block.php [hunk 1]
```diff
@@ -32,13 +32,11 @@
 	require_once PIXALB_DATE_BLOCK_PATH . 'vendor/autoload.php';
 }
 
-$updater = PucFactory::buildUpdateChecker(
+PucFactory::buildUpdateChecker(
```
Comment (szepeviktor) [inline date-block.php:35]: Remove a global variable.

File: date-block.php [hunk 2]
```diff
@@ -47,7 +45,7 @@
  *
  * @see https://developer.wordpress.org/reference/functions/register_block_type/
  */
-function date_block_init() {
+function init() {
```
Comment (szepeviktor) [inline date-block.php:48]: We are already in a namespace!

File: date-block.php [hunk 3]
```diff
@@ -67,14 +65,14 @@ function date_block_init() {
...
  * Registers the block textdomain.
  *
  * @return void
  */
-function date_block_i18n() {
+function i18n() {
 	load_plugin_textdomain( 'date-block', false, plugin_basename( PIXALB_DATE_BLOCK_PATH ) . '/languages' );
 }
-add_action( 'plugins_loaded', __NAMESPACE__ . '\date_block_i18n' );
+add_action( 'init', __NAMESPACE__ . '\i18n' );
```
Comment (szepeviktor) [inline date-block.php:78]: `load_plugin_textdomain` should not be called until `init`.
JIT: https://make.wordpress.org/core/2016/07/06/i18n-improvements-in-4-6/

**Comment by s3rgiosan** [comment]
Thanks @szepeviktor

**Review by s3rgiosan — APPROVED** [review]

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**